tools/data-migration-overview.md

```diff
@@ -41,3 +41,29 @@ dmctl is the command line tool used to control the DM cluster.
 - Checking the state of data synchronization tasks
 - Handling the errors during data synchronization tasks
 - Verifying the configuration correctness of data synchronization tasks
+
+## Data synchronization introduction
+
+This section describes the data synchronization feature provided by Data Migration in detail.
+
+### Black and white lists synchronization at the schema and table levels
+
+The black and white lists filtering rule of the upstream database instances is similar to MySQL replication-rules-db/tables, which can be used to filter or only synchronize all operations of some databases or some tables.
+
+### Binlog event filtering
+
+Binlog event filtering is a more fine-grained filtering rule than the black and white lists filtering rule at the schema and table levels. You can specify the Binlog events of the database(s) or table(s) that you need to synchronize or filter out using statements like `INSERT`, `TRUNCATE TABLE`.
```

Comment:
`using statements like ...` is a little ambiguous, is it `You can specify ... using statements like ...` or `Binlog events using statements like...`